tests/test_core.py

```diff
@@ -1104,6 +1099,32 @@ def test_delimited_tuple_passed_invalid_type(web_request, parser):
     assert excinfo.value.messages == {"json": {"ids": ["Not a valid delimited tuple."]}}
 
 
+def test_delimited_list_custom_empty_value(web_request, parser):
+    web_request.json = {"ids": "1,,3"}
+    schema_cls = Schema.from_dict({"ids": fields.DelimitedList(fields.Int(), empty=0)})
+    schema = schema_cls()
+
+    parsed = parser.parse(schema, web_request)
+    assert parsed["ids"] == [1, 0, 3]
+
+    data = schema.dump(parsed)
+    assert data["ids"] == "1,0,3"
```

Comment:
Not a blocker at all but do we need to test `dump`?

Reply:
I'm not overly attached to checking it...

It keeps the tests uniform with other delimited field tests, but I'd say it's not a valuable check in these cases. Why don't I remove it? No point keeping it if we agree that it's not doing anything.

Reply:
I didn't realize other tests did this. I'm fine both ways.

Reply:
I removed it. Once it was pointed out, it started to bother me. :sweat_smile: